guide/14-deep-learning/leveraging_llms_for_real_time_decision_making_in_calamities.ipynb

```diff
@@ -0,0 +1,1042 @@
+{
```

Comment:
<p>Model extension requires the process be wrapped in a class with the following functions implemented:</p><ul><li><code>__init__</code></li><li><br></li><li><code>getParameterInfo</code></li><li><br></li><li><code>initialize</code></li><li><br></li><li><code>getConfiguration</code></li><li><br></li><li><code>predict</code></li></ul><p>Let’s create a custom extension model to classify input tweets into the categories:&nbsp;<strong>Health Emergency, Public Service Failures, and Other</strong>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>updated</p>